mlir/test/xmir/mobilenetv1.mid.mlir

```diff
@@ -6,20 +6,22 @@ module {
 
 // CHECK: func @mobilenetv1(%arg0: memref<1x224x224x3xf32>, %arg1: memref<32x3x3x3xf32>, %arg2: memref<3x3x32x1xf32>, %arg3: memref<64x1x1x32xf32>, %arg4: memref<3x3x64x1xf32>, %arg5: memref<128x1x1x64xf32>, %arg6: memref<1x56x56x128xf32>) {
 // CHECK:   %token = async.launch @mobilenetv1_outlined_part_0 (%arg0, %arg1, %{{.*}}) : (memref<1x224x224x3xf32>, memref<32x3x3x3xf32>, memref<1x112x112x32xf32>)
-// CHECK:   async.await %token : !async.token
-
-// CHECK:   %token_0 = async.launch @mobilenetv1_outlined_part_1 (%{{.*}}, %arg3, %{{.*}}) : (memref<1x112x112x32xf32>, memref<64x1x1x32xf32>, memref<1x112x112x64xf32>)
-// CHECK:   async.await %token_0 : !async.token
-        
-// CHECK:   %token_1 = async.launch @mobilenetv1_outlined_part_2 (%{{.*}}, %arg5, %arg6) : (memref<1x56x56x64xf32>, memref<128x1x1x64xf32>, memref<1x56x56x128xf32>)
-// CHECK:   async.await %token_1 : !async.token
-    
+
+// CHECK:   %token_0 = async.launch @mobilenetv1_outlined_part_1 [%token] (%{{.*}}, %arg2, %{{.*}}) : (memref<1x112x112x32xf32>, memref<3x3x32x1xf32>, memref<1x112x112x32xf32>)
```

Comment:
With the changed defaults, there are more outlined kernels and thus more launches.  I don't know why the original version had awaits after each call when the newer version doesn't.  These are changes to make the tests pass, and may not match the spirit of the original test.

Reply:
@krzysz00 that happens in miopen-async-launch. The tosa-partition pass just does the outlining with tensor sequential dataflow.